AY2324S1-CS2103T-F12-4/tp
# Edit meeting command

Edit meeting command added with similar rules as existing edit contact command
- Meeting index used to denote which meeting to edit
- Prefixes are used to denote which field to edit
- All prefixes are optional,  at least 1 prefix must be present to use the command
- Tags can be removed with just "t/", multiple tags can be added with "t/TAG_1 t/TAG_2"
- Test cases for edit meeting

Other changes
- Status prefix for contacts changed from "status/" to "s/"
- View meeting will automatically update display when meetings are edited/deleted
- Add contact command usage message edited, square brackets used to denote Status and Remark as optional
- Meeting Time message constraints edited to be more helpful
- App window now labelled as OutBook

UserGuide
- Edited entry for viewm, viewc, editm
- Slight update to addmc and rmmc since all arguments are called by index (someone can double check)
